right values.cpp

```diff
@@ -0,0 +1,21 @@
+
+// Unit test, right values

```

Comment:
there is no test - just a sceletone.

Reply:
Sorry, I have exidently added the wrong file. 

I have fixed it now.

Reply:
Here is a commit for that fix: https://github.com/ViktoriiaPartyka/csdt-2122-ki48-ViktoriiaPartyka/pull/3/commits/53480dde4e53ba144f3c4bf59bba03870a221a43

Reply:
I do not agree but I have no time to argue with you.